eng/pipelines/runtime.yml

```diff
@@ -479,6 +479,27 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_mono.containsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+#
+# Build Mono release with LLVM AOT
+# Only when libraries, mono, or the runtime tests changed
+# Currently only these architectures are needed for the runtime tests.
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/mono/templates/build-job.yml
+    runtimeFlavor: mono
+    buildConfig: release
+    platforms:
+    - Linux_x64
+    jobParameters:
+      runtimeVariant: llvmaot
+      condition: >-
+        or(
+          eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
```

Comment:
Why do we have to build mono AOT when libraries change?

Reply:
Duplicated from the non-LLVM AOT template instantiation directly above. :)

Reply:
The goal was to run Mono lanes when there are Library changes, Mono changes and test changes. We have moved over to .NET BCL, so any change there can affect Mono, and we should test it out. (We just had a perf regression creep in due to a change in libraries a couple weeks ago !). I would recommend leaving the change as is, and see how badly this affects the queues/wait times for CI.

Reply:
It doesn't make sense to run runtime tests when libraries changes, libraries are not even used in that scope.

When libraries changes we run managed (libraries) tests using a mono runtime, that I agree 😄